Documentation/ExtensionArchitecture/HowTo/CreateNewExtension.rst

```diff
@@ -29,12 +29,9 @@ Installing the newly created extension
 Starting with TYPO3 v11 it is no longer possible to install extensions in TYPO3
 without using Composer in Composer-based installations.
 
-However during development it is likely that you will want to test your extension locally
-before publishing it.
-
-During development, place the extension in a directory called,
-:file:`packages` in TYPO3s root directory. You can name is directory
-however you choose.
+However during development it is necessary to test your extension locally
+before publishing it. Place the extension directory in a directory called,
+:file:`packages` in TYPO3's root directory. You can freely name the extension directory.
```

Comment:
I know, it's the text of the original, but what about `TYPO3 project`? It makes it more clear, that the directory should not be created somewhere in `vendor/typo3/` or `public/typo3/`

```suggestion
:file:`packages` in TYPO3 projects root directory. You can freely name the extension directory.
```